stories/patterns/ux-patterns/form/docs/examples/checkbox-and-radio.tsx

```diff
@@ -0,0 +1,145 @@
+    reset({ export: 'first' });
+    setSelectedFirst(value);
+  };
+  const onChangeCheckbox = (
+    checked: boolean,
+    e?: React.SyntheticEvent<HTMLInputElement, Event>,
+  ) => {
+    const value = e?.currentTarget.value as string;
+
+    const tmpArray = checked ? [...selectedValue, value] : selectedValue.filter((v) => v !== value);
+    tmpArray.length && reset({ export: 'selected' });
+    setSelectedValue(tmpArray);
+  };
+
+  return (
+    <Flex tag='form' onSubmit={handleSubmit(onSubmit)} direction='column' alignItems='flex-start'>
+      <Flex direction='column' mb={4}>
+        <Text size={300} tag='label' mb={4}>
+          Export data
+        </Text>
+        <Controller
+          render={({ field: { value, onChange }, ...props }) => {
+            return (
+              <RadioGroup {...props} value={value} onChange={(_v, e) => onChange(e)} size='l'>
+                <Radio mb={3}>
+                  <Radio.Value value='all' />
+                  <Radio.Text>All</Radio.Text>
+                </Radio>
+                <Radio mb={3}>
+                  <Radio.Value value='selected' />
+                  <Radio.Text>Selected</Radio.Text>
+                  {value === 'selected' &&
+                    [100, 500].map((v) => (
+                      <Checkbox
+                        size='l'
+                        ml={2}
+                        key={v}
+                        state={
+                          value.includes('selected') && errors['export'] ? 'invalid' : 'normal'
+                        }
+                      >
+                        <Checkbox.Value value={v} onChange={onChangeCheckbox} />
+                        <Checkbox.Text children={v} />
+                      </Checkbox>
+                    ))}
+                </Radio>
+                <Radio style={{ alignItems: 'center' }}>
+                  <Radio.Value value='first' />
+                  <Radio.Text>First</Radio.Text>
+
+                  <Tooltip>
+                    <Tooltip.Popper
+                      id='form-select-error'
+                      theme='warning'
+                      placement='top'
+                      visible={selectInFocus && value.includes('first') && !!errors['export']}
+                    >
+                      Field is required.
+                    </Tooltip.Popper>
+                    <Tooltip.Trigger
+                      w={'100%'}
+                      inline={false}
+                      onFocus={() => setSelectInFocus(true)}
+                      onBlur={() => setSelectInFocus(false)}
+                    >
+                      <Select
+                        size='l'
+                        ml={2}
+                        state={value.includes('first') && errors['export'] ? 'invalid' : 'normal'}
+                        tag={ButtonTrigger}
+                        options={optionsFirst}
+                        onChange={onChangeSelect}
+                      />
```

Comment:
I think here 
`<Tooltip.Trigger><Select>` will be errors with accessibility

Reply:
Removed